source/Cosmos.System2/Graphics/Canvas.cs

```diff
@@ -904,15 +902,15 @@ public virtual void DrawRectangle(Pen pen, float x_start, float y_start, float w
         /// <exception cref="ArgumentOutOfRangeException">Thrown on fatal error.</exception>
         public virtual void DrawImage(Image image, int x, int y)
         {
-            Pen _pen = new Pen(Color.White);
+            Color _color = Color.White;
```

Comment:
Unnecessary assignment to _color; the value of `Color.White` will always be replaced with `(Color.FromArgb(image.rawData[_x + _y * image.Width]));`